app/assets/javascripts/src/Metamaps.Visualize.js

```diff
@@ -143,6 +143,9 @@ Metamaps.Visualize = {
       self.mGraph.graph.empty()
     }
 
+
+    if (self.type == 'ForceDirected' && Metamaps.Active.Mapper) $.post('/maps/' + Metamaps.Active.Map.id + '/events/user_presence')
```

Comment:
we should have a better way to say "Map View" or "Topic View"

Maybe something in Metamaps.Settings or Metamaps.Router?

Are there lots of spots in the code that check self.type == ForceDirected?